pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -284,15 +291,32 @@ def _pokemon_matches_config(self, config, pokemon, default_logic='and'):
         if pokemon.cp > catch_cp:
             catch_results['cp'] = True
 
+        catch_below_cp = pokemon_config.get('catch_below_cp', 12000)
+        if pokemon.cp < catch_below_cp:
+            catch_results['cp'] = True
+
         catch_iv = pokemon_config.get('catch_above_iv', 0.8)
         if pokemon.iv > catch_iv:
             catch_results['iv'] = True
 
+
+        catch_results['fa'] = ( len(pokemon_config.get('fast_attack', [])) == 0 or unicode(pokemon.fast_attack) in map(lambda x: unicode(x), pokemon_config.get('fast_attack', [])))
```

Comment:
You do not need all the Unicode things. If you want to manipulates string as Unicode, just add `from __future__ import unicode_literals` at the start of the file.